architecture-building-systems/CityEnergyAnalyst
# I2120 re(making)factor optimization

This is the angriest branch of CEA part 1. 

By no means this is the end version, we are far from that. I just would like to merge before this becomes way to big.

@shanshanhsieh please test it in the following days and merge.

Only the heating case will work for now.  the cooling_workflow will need to be reinvented (several problematic issues in terms of theory and implementation exist there)

How to test:
1. take this case study. https://we.tl/t-DCZNNAPm0u
2. use the weather file of zurich or zug.
2. run optimization_main for district heating = True, district_cooling=False. for a few generations
3. Check the graphs inside plots_optimization and plots/supply_system
4. update
